test/mock/createAPIResponse.ts

```diff
@@ -1,21 +1,41 @@
 import { SearchForFacetValues, Response, MultiResponse } from 'algoliasearch';
 
-export const createSingleSearchResponse = (
-  args: Partial<Response> = {}
-): Response => ({
-  hits: [],
-  nbHits: 0,
-  page: 0,
-  nbPages: 0,
-  hitsPerPage: 4,
-  processingTimeMS: 1,
-  query: '',
-  params: '',
-  index: 'index_name',
-  ...args,
-});
+export function createSingleSearchResponse<THit = any>(
```

Comment:
You can keep the function expression signature with generic:

```ts
export const createSingleSearchResponse = <THit = any>(
  subset: Partial<Response<THit>> = {}
): Response<THit> => {
  // ...
}
```

Reply:
was trying to find how to write it, but decided function was also fine, thanks!